arangod/Aql/DocumentProducingNode.cpp

```diff
@@ -32,6 +32,8 @@
 #include "Aql/Variable.h"
 #include "Basics/StaticStrings.h"
 #include "Basics/VelocyPackHelper.h"
+#include "OptimizerUtils.h"
```

Comment:
To be consistent with the other includes, this could use `Aql/` and be moved up so the includes are sorted alphabetically.
```suggestion
#include "Aql/OptimizerUtils.h"
```